crates/ln-dlc-node/src/node/dlc_channel.rs

```diff
@@ -135,6 +140,57 @@ where
         Ok(dlc_channel)
     }
 
+    #[autometrics]
+    pub fn get_temporary_contract_id_by_sub_channel_id(
+        &self,
+        sub_channel_id: ChannelId,
+    ) -> Result<ContractId> {
+        let dlc_channel_id = self
+            .dlc_manager
+            .get_store()
+            .get_sub_channel(sub_channel_id)?
+            .context("not found")?
+            .get_dlc_channel_id(0)
```

Comment:
❓ I guess this is because we only ever have one position?

Maybe we could add a `debug_assert!` here to ensure there aren't any other dlc_channels.

Reply:
This is the thread where Tibo explained the state of affairs: https://github.com/p2pderivatives/rust-dlc/pull/116#discussion_r1255128516.